packages/zowe-explorer/README.md

```diff
@@ -54,7 +54,7 @@ For more information, see [Changelog](https://marketplace.visualstudio.com/items
 
 ## Prerequisites
 
-- Install [Node.js](https://nodejs.org/en/download/) v8.0 or later.
+- Install [Node.js](https://nodejs.org/en/download/) v12.0 or later.
```

Comment:
I was just changing the version already mentioned but if this isn't a Prerequisite any more for Zowe Explorer to work in VS Code should we just remove it?

Reply:
Thank you, Billie! If it's not a pre-req anymore, we surely need to remove it then

Reply:
Do we need to tag here someone who can confirm this?

Reply:
I can test it out and confirm

Reply:
it is confirmed that nodejs isn't a pre-req for using VS Code with Zowe Explorer so I will remove this

Reply:
I will leave it as a pre-req in the top level README since it is required for developers and such but removed it from the Zowe Explorer README that shows in VS Code.